### PR TITLE
Origin startup stdout cleanup - Incorporating Goose and build info output into the logging system

### PR DESCRIPTION
**Pull request by Saartank**
This PR fixes issue #1502 

- It incorporates Goose output into the logging system. Goose has two levels of output: `info` and `fatal`, which are now mapped to the corresponding levels in `logrus`.
- The build information at server startup is now logged as part of the `debug` logs.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -919,6 +931,13 @@ func PrintPelicanVersion(out *os.File) {
 	fmt.Fprintln(out, "Built By:", GetBuiltBy())
 }
 
+func LogPelicanVersion() {
+	log.Debugf("Version: %s", GetVersion())
```
Comment (bbockelm) [inline config/config.go:935]: If all we're doing is concatenating strings together, the `Debugln` variant is more elegant.  Additionally given the information used to be printed all the time by default; let's move it to `Info`:

```suggestion
	log.Infoln("Version:", GetVersion())
```

(and the subsequent lines as well.)

**Review by bbockelm — CHANGES_REQUESTED** [review]
Just one small comment, then I think we're ready to go.

**Comment by bbockelm** [comment]
@Saartank - looks like there's still a CI failure; can you investigate?

Also, once done, the commit history is a bit scattered on this one.  Please squash it to one commit per piece of functionality.

**Comment by Saartank** [comment]
Closing this PR because I made a mistake while changing the commit history. Created a new PR with a clean commit history: [https://github.com/PelicanPlatform/pelican/pull/1639](https://github.com/PelicanPlatform/pelican/pull/1639).

**Pull request closed without merging**